include/deal.II/base/std_cxx11/type_traits.h

```diff
@@ -33,6 +33,8 @@ namespace std_cxx11
   using std::is_standard_layout;
   using std::is_trivial;
   using std::enable_if;
+  using std::true_type;
```

Comment:
As mentioned in the email to the dev@ mailing list, no need for adjusting copyright years -- we have a script for that.